### PR TITLE
Clean up R API docs.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: docs/source/R-api.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ R API
 
 The MLflow R API allows you to use MLflow :doc:`Tracking <tracking/>`, :doc:`Projects <projects/>` and :doc:`Models <models/>`.
 
-For instance, you can use the R API to `install MLflow`_, start the `user interface <MLflow user interface_>`_, `create <Create Experiment_>`_ and `list experiments`_, `save models <Save Model for MLflow_>`_, `run projects <Run in MLflow_>`_ and `serve models <Serve an RFunc MLflow Model_>`_ among many other functions available in the R API.
+You can use the R API to `install MLflow`_, start the `user interface <Run MLflow user interface_>`_, `create <Create Experiment_>`_ and `list experiments`_, `save models <Save Model for MLflow_>`_, `run projects <Run in MLflow_>`_ and `serve models <Serve an RFunc MLflow Model_>`_ among many other functions available in the R API.
```
Comment (aarondav) [inline docs/source/R-api.rst]: Should we link `list experiments` too? I think `<List Experiments`> should be a valid link.

**Comment by aarondav** [comment]
LGTM, thanks @stbof. @javierluraschi PTAL if you get a chance.

File: docs/source/R-api.rst (hunk 1 above)
Comment (stbof) [reply docs/source/R-api.rst]: Good catch. Added link.

**Comment by javierluraschi** [comment]
This file gets generated from the `.R` doc files, so we rather need to update the `.R` files and then regenerate this file by running `https://github.com/mlflow/mlflow/blob/master/mlflow/R/mlflow/document.R`, otherwise the R help won't be up to date.

**Comment by javierluraschi** [comment]
@stbof  I can help regenerate the docs later, but would you mind making the changes in the R files as well? Would be much appreciated!

For instance, the first edit `For instance, you can use the R API` would go to [mlflow/R/mlflow/document.R#L40](https://github.com/mlflow/mlflow/blob/1e01ed05e202622dcb2ea623dd794eb49a9d6ac1/mlflow/R/mlflow/document.R#L40)

The change `Executes a generic MLflow command through the commmand line interface`, would go in [mlflow/R/mlflow/R/cli.R#L3](https://github.com/mlflow/mlflow/blob/d07d25393f625ed29ac941056256ea7aab497307/mlflow/R/mlflow/R/cli.R#L3), etc.

**Comment by stbof** [comment]
Will get back to this later this week.

**Comment by stbof** [comment]
@javierluraschi  Moved as requested.

**Comment by mateiz** [comment]
@javierluraschi and @stbof , any updates on this PR? We should update the docs but this is falling out of sync with other changes.

**Comment by stbof** [comment]
I moved my edits to the .R files as requested by @javierluraschi. Waiting for him to approve and merge.

**Comment by javierluraschi** [comment]
Looks good to merge, thanks for updating the R docs!

**Pull request merged**